core/arch/arm/kernel/secure_partition.c

```diff
@@ -1175,6 +1254,12 @@ static TEE_Result fip_sp_map_all(void)
 		}
 	}
 
+	if (res) {
```

Comment:
This is never true, earlier error case returned straight.

Reply:
Ack